presentation-core/src/main/java/tachiyomi/presentation/core/components/WheelPicker.kt

```diff
@@ -166,6 +166,18 @@ private fun <T> WheelPicker(
                             showManualInput = false
                         }
                     },
+                onKeyboardAction = {
+                    scope.launch {
+                        items
+                            .indexOfFirst { it.toString() == value.text }
+                            .takeIf { it >= 0 }
+                            ?.apply {
+                                internalOnSelectionChanged(this)
+                                lazyListState.scrollToItem(this)
+                            }
+                        showManualInput = false
+                    }
```

Comment:
since this is just a copy of the stuff above make this a lambda function and apply to both